packages/ilios-common/addon/classes/events-base.js

```diff
@@ -1,8 +1,63 @@
 import Service from '@ember/service';
 import { DateTime } from 'luxon';
 import { mapBy, sortBy, uniqueValues } from 'ilios-common/utils/array-helpers';
+import { getSlug } from 'ilios-common/utils/event-helpers';
 
 export default class EventsBase extends Service {
+  /**
+   * Proxy handler for events.
+   */
+  handler = {
```

Comment:
the proxy handler. 

i wonder what this does for performance, as the "intercepted attrs" are recalculated on each read. 

a cached getter would probably be better here, but cannot be implemented.

as another aside - the native Proxy class cannot be sub-classed :clown_face: .